docs/source/common_issues.rst

```diff
@@ -197,15 +201,16 @@ unexpected errors when combined with type inference. For example:
 
    lst = [A(), A()]  # Inferred type is List[A]
    new_lst = [B(), B()]  # inferred type is List[B]
-   lst = new_lst  # mypy will complain about this, because List is invariant
+   lst = new_lst  # error: Incompatible types in assignment (expression
```

Comment:
I think it's worth mentioning that mypy complains about the assignment because of the fact that `List` is invariant.